package.json

```diff
@@ -16,6 +16,8 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@bazel/bazel": "^0.18.1"
+    "@bazel/bazel": "^0.18.1",
+    "gulp-postcss": "^10.0.0",
+    "postcss": "^8.4.45"
```

Comment:
I don't think we should need new packages?

Reply:
We don't